elements/rh-secondary-nav/demo/rh-secondary-nav.html

```diff
@@ -3,168 +3,157 @@
 <script type="module" src="rh-secondary-nav.js"></script>
 
 <div class="mock-nav"></div>
-<rh-secondary-nav role="navigation">
+<rh-secondary-nav role="navigation" data-analytics-region="secondary nav">
   <a href="#" slot="logo" id="logo-id">
     Red Hat Ansible Automation Platform
   </a>
   <ul slot="nav" aria-labelledby="logo-id">
     <li>
       <rh-secondary-nav-dropdown>
-        <a href="#" slot="link">Explore</a>
+        <a href="#" data-analytics-level="1" data-analytics-text="Explore" slot="link">Explore</a>
```

Comment:
I'm not sure if *-level="" is still being used, but we can leave it for now. Rickey (SDI) and Marcia can review and verify. No action needed.

Reply:
Got it. Yeah Allen had mentioned that the main nav uses them, but the footer does not, so he wasn't sure.